wikidata/entity.py

```diff
@@ -237,25 +237,28 @@ def attributes(self) -> Mapping[str, object]:
     def load(self) -> None:
         url = './wiki/Special:EntityData/{}.json'.format(self.id)
         result = self.client.request(url)
-        assert isinstance(result, collections.abc.Mapping)
-        entities = result['entities']
-        assert isinstance(entities, collections.abc.Mapping)
-        assert len(entities) == 1
-        redirected = False
-        entity_id = self.id
-        try:
-            data = entities[entity_id]
-        except KeyError:
-            entity_id = cast(EntityId, next(iter(entities)))
-            data = entities[entity_id]
-            redirected = True
-        assert isinstance(data, collections.abc.Mapping)
-        self.data = data
-        self.id = entity_id
-        if redirected:
-            canon = self.client.get(entity_id, load=False)
-            if canon.data is None:
-                canon.data = dict(data)
+        if result is not None:
```

Comment:
It could be more readable if it [returns early][1].  (Or at least, it's more maintainable to me.)

[1]: http://www.itamarweiss.com/personal/2018/02/28/return-early-pattern.html

Reply:
I made it return early.